app/models/asset.rb

```diff
@@ -87,10 +89,9 @@ class Asset
   end
 
   def accessible_by?(user)
-    return true unless draft?
-    return true if access_limited.empty?
+    return true unless draft? && access_limited?
```

Comment:
I think the `access_limited?` question needs to be inverted here - it's accessible if it's *not* access limited :-).

Reply:
I think the logic is right - unless (i.e. if not) `draft` AND `access limited` then return true

Reply:
Oops - smug logic smile vanquished :-|.

Reply:
@benthorner this brings back a memory 😄 I stand by "unless" and an "and" read wrong 😉